test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -92,7 +98,7 @@ func InitialSetupWithTeam(t *testing.T) KubernetesConfigGenerateProjectSuite {
 	s.generator = newAtlasE2ETestGenerator(s.t)
 	s.generator.generateTeam("Kubernetes")
 	s.generator.generateEmptyProject(fmt.Sprintf("Kubernetes-%s", s.generator.projectName))
-	s.expectedProject = referenceProject(s.generator.projectName, targetNamespace)
+	s.expectedProject = referenceProject(s.generator.projectName, targetNamespace, expectedLabels)
```

Comment:
Maybe we can go with option of removing label instead of creating one?

Reply:
I think we can fix it in follow up PR.

Reply:
With the "CRD versions" feature there always will be labels, so, I'd say we add and validate them instead of removing. Also, tests can be extended to verify previous version of the operator